optimum/exporters/ipex/modeling/xpu/xpu_modeling_llama.py

```diff
@@ -0,0 +1,127 @@
+from typing import Tuple
+import torch
+import torch.nn as nn
+from typing import Optional
+
+
+import intel_extension_for_pytorch
+from intel_extension_for_pytorch.transformers.models.xpu.optimize_transformers.modules.llama import NewIPEXLLAMABlock
```

Comment:
Any model-level changes should be in the optimum-intel as they can be aligned with the transformers version. Only Op level changes can be imported from ipex.